apis/kubedb/v1alpha1/types.go

```diff
@@ -81,6 +81,7 @@ type LocalSpec struct {
 
 type S3Spec struct {
 	Endpoint string `json:"endpoint,omitempty"`
+	Region   string `json:"region,omitempty"`
```

Comment:
Why do we need this?

Reply:
Without Region, Stow returns error: access denied: Bucket Location. Something like this.